public/request/comment/delete.php

```diff
@@ -8,8 +8,9 @@
 $response = [];
 $response['ArtID'] = $articleID;
 $response['CommentID'] = $commentID;
-if (RA_ReadCookieCredentials($user, $points, $truePoints, $unreadMessageCount, $permissions, \RA\Permissions::Registered)) {
-    $response['Success'] = RemoveComment($articleID, $commentID);
+$isAdmin = RA_ReadCookieCredentials($user, $points, $truePoints, $unreadMessageCount, $permissions, \RA\Permissions::Admin);
```

Comment:
This `RA_ReadCookieCredentials` call in unnecessary. You can leave the other one and then use the `$permissions` output parameter to decide whether the user is an admin: `$permissions >= \RA\Permissions::Admin`.

Reply:
Yes, it's already fixed ;)